staging/src/k8s.io/kube-scheduler/config/v1alpha2/types.go

```diff
@@ -70,11 +70,6 @@ type KubeSchedulerConfiguration struct {
 	// nodes will be scored.
 	PercentageOfNodesToScore *int32 `json:"percentageOfNodesToScore,omitempty"`
 
-	// Duration to wait for a binding operation to complete before timing out
-	// Value must be non-negative integer. The value zero indicates no waiting.
-	// If this value is nil, the default value will be used.
-	BindTimeoutSeconds *int64 `json:"bindTimeoutSeconds"`
```

Comment:
You can't do this anymore... we have to introduce v1alpha3

But we could introduce v1beta1 instead. cc @ahg-g @Huang-Wei

Reply:
Yes, we need to get ComponentConfig to beta before you can remove this. We should do that in 1.19

Reply:
yes, [my plan](https://github.com/kubernetes/kubernetes/pull/83726#issuecomment-596056943) is to update the code when the new version is ready. I can send out a PR like #87628 to introduce a v1alpha3 or v1beta1 if you agree.

Reply:
Not sure if we need a KEP for graduating to beta. @ahg-g?

Reply:
I think we need a KEP for graduating to Beta, but will likely be something simple.

Reply:
Also, before doing that, I would like to do #89260 first

Reply:
> Let's put a TODO here to indicate that we shouldn't copy this field into new APIs.

how about we avoid an api review in this PR, lets just separate the issue of removing the parameter from migrating functionality to plugins.

Reply:
sg. I would appreciate if @cofyc can still do the follow up PR though

Reply:
sure, I'll do the follow-up PR. I'll update this PR soon. API change will be addressed in the next PR.